contracts/libraries/SafeToL2Setup.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.7.0 <0.9.0;
+
+import {SafeStorage} from "../libraries/SafeStorage.sol";
+
+/**
+ * @title Safe to L2 Setup Contract
+ * @dev We made the L1 "default" because on average the L2 price per gas is ~1000x cheaper than L1, so defaulting to L1 and going to L2 is overall more efficient.
```

Comment:
Nit: the 1000x cheaper was specific to OP at that moment and not generally true of L2s. I would be more generic here:

```suggestion
 * @dev This contract expects the singleton to be the {Safe} by default. Even if there are more
 * {SafeL2} proxies deployed, the average gas cost on L2s is significantly lower, making the
 * current design more economically efficient overall.
```